docs/sdks/debugging.mdx

```diff
@@ -117,3 +133,14 @@ Android: Available with  `Statsig.openDebugView()`. Available in [v4.29.0](https
 |Landing|Gates List|Gate Details|Experiment Details|
 |--|--|--|--|
 |![client-debugger-landing](https://github.com/statsig-io/statsig-sdk-debugger-chrome-extension/assets/95646168/fa6d7237-eb47-4f09-896c-696cfd5c956c)|![client-debugger-gates-list](https://github.com/statsig-io/statsig-sdk-debugger-chrome-extension/assets/95646168/161d8f35-a9b8-4ff9-b549-e04d04acac8a)|![client-debugger-gate-info](https://github.com/statsig-io/statsig-sdk-debugger-chrome-extension/assets/95646168/ab15e586-5259-4475-8f5c-018b2ab6e8db)|![client-debugger-experiment-details](https://github.com/statsig-io/statsig-sdk-debugger-chrome-extension/assets/95646168/920a6e8a-eb84-4d37-bf77-bb909a575d58)|
+
+
+## SDK Event Handling and Minimizing Event Loss
+
+The SDK batches and flushes events in the background to our server. When the volume of incoming events exceeds the SDK's flushing capacity, some events may be dropped after a certain number of retries. To reduce the chances of event loss, you can adjust several settings in the Statsig options:
```

Comment:
this *only* applies to python at the moment.  Id document it in the python sdk docs instead of in a generic debugging section.  If you do leave it here, then it should specify which sdk type/version it applies to